### DISCUSSION TITLE
Custom trace id

### DISCUSSION THREAD

**Discussion by R4cOOn** [getsentry/support#34] [Support] [answered]
We have a custom trace id 
![image](https://user-images.githubusercontent.com/1311219/177748306-fef14275-4814-4feb-a32e-f0d1aed26247.png)
as well as a transaction id 
![image](https://user-images.githubusercontent.com/1311219/177755036-31e6ccb7-dbdd-4b24-879a-cf4f0a3f457c.png)
 and a span id 
![image](https://user-images.githubusercontent.com/1311219/177755124-ea7e1984-4019-4348-8a0b-982f4e5f85d1.png)

How can we setup Sentry to use all these for error reporting?  
We would like to have errors grouped or merged together per trace id as the same error is logged several times as it bubbles up through the different application layers.

**Comment by souredoutlook** [comment, marked as answer]
Hey @R4cOOn we've rolled out some new processes (for example, transferring ownership of support-like discussions to the Support team) let me know if you still need a hand with this as it's admittedly been far too long since we've addressed your question.

At first glance I'd say what you're looking for isn't possible as grouping across projects is not supported, but I may have misunderstood this statement:

> is logged several times as it bubbles up through the different application layers

If these layers all report into the same Sentry project it would be possible to [use server side fingerprinting](https://docs.sentry.io/product/data-management-settings/event-grouping/fingerprint-rules/) to group these events together but this would also result in a very large number of Issue groups, to the point where grouping would likely be meaningless. 

Simply setting all of these fields as tags would allow you to use [Discover](https://github.com/getsentry/sentry/issues/39231) to correlate the tags across events (and projects) with a single search.
